docs/cookbook.rst

```diff
@@ -27,7 +29,9 @@ Using MakeNote
 You can also retrieve initial answers
 with the :class:`~screenpy.actions.MakeNote` class,
 and retrieve the value
-with a :ref:`direction <directions>`::
+with a :ref:`direction <directions>`
```

Comment:
I think i like replacing the `::`s with a `.` in the text and the `.. code-block:: python` to be more explicit.

However, you missed a `.` here. :P

Reply:
fixed